spec/models/achievement_spec.rb

```diff
@@ -39,6 +39,17 @@
     end
   end
 
+  describe "to_s" do
+    it "is well-formed" do
+      subject.side = 'corp'
+      subject.title = 'Membrane'
+      subject.points = 2
+      subject.description = 'Whatever'
```

Comment:
[Huh.](https://viget.com/extend/just-use-double-quoted-ruby-strings)

I come from a JavaScript background where single quotes are preferred. In languages with string interpolation I default to using single quotes to indicate that there is no interpolation going on.

Happy to change it to respect your style Johno... assuming this is your style and not Hound just moaning.